subcommand/server-acl-init/policies_test.go

```diff
@@ -0,0 +1,289 @@
+package serveraclinit
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestAgentPolicy(t *testing.T) {
```

Comment:
the function you're testing is `rules`. I think `policy` is a better name but either way they should be consistent between the tests and the method.

Reply:
Switched to rules.